css/nmcstyle.scss

```diff
@@ -36,6 +36,7 @@
 @import 'layouts/status.scss';
 @import 'layouts/conflictdialog.scss';
 @import 'layouts/guestview.scss';
+@import 'layouts/search.scss';
```

Comment:
Is search a layout or a component?

Reply:
Moved to the components folder as it makes more sense, thanks.